src/mobile/setup.md

```diff
@@ -0,0 +1,36 @@
+# Development
```

Comment:
Also add the prerequisites - 
1. Android SDK
2. JAVA

Also add how to generate avd for android emulator after installing SDK -

```
sdkmanager --install 'system-images;android-29;google_apis_playstore;x86_64'
sdkmanager --install 'extras;intel;Hardware_Accelerated_Execution_Manager'

.\avdmanager.bat create avd -f -n crimsonAVD -d 17 -b google_apis/x86_64 --package 'system-images;android-29;google_apis;x86_64'
```

Reply:
And how to run the emulator - 

```
emulator -avd crimsonAVD
```